.rubocop.yml

```diff
@@ -3,3 +3,7 @@ Documentation:
 
 LineLength:
   Max: 99
+
+Metrics/BlockLength:
+  Exclude:
+    - spec/**/*
```

Comment:
rememeber to have a new line at the end of the file

Reply:
ready!